quick_start/more.rst

```diff
@@ -0,0 +1,278 @@
+
+.. code :: java
+  
+    def speciesLength = irisclassify.predict | demo.numChars
+
+To build workflows like this, we create them as modules in a similar way we created a Python module -- with ``init``. Let's create a new directory called ``irisworkflow``, ``cd`` into it, and create a new workflow module.
+
+.. code :: bash
+  
+    ~/ $ mkdir irisworkflow; cd irisworkflow
+    ~/irisworkflow/ $ nstack init workflow 
+    Workflow module 'irisworkflow' successfully initialised at /var/home/fedora/irisworkflow
+
+Instead of creating an ``nstack.yaml``, this creates a single file, ``workflow.nml``, which contains our workflow module.
+
+.. code :: java
+  
+  module irisworkflow {
+    // A sample workflow
+    def w = source(http:///s : Text) | Module1.numChars | sink(log:// : Integer)
+  }
+
+You will notice that the module itself is named ``irisworkflow`` after the directory name, and has an example workflow in it, ``w``. We're going to replace this with our ``speciesLength`` workflow above.
+
+.. code :: java
+  
+  module irisworkflow {
+    // A sample workflow
+    def speciesLength = irisclassify.predict | demo.numChars
+  } 
+
+As with others modules, we can now build ``irisworkflow`` with the ``build`` command:
+
+.. code :: bash
+ 
+  ~/irisworkflow/ $ nstack build
+  Building NStack Workflow module irisworkflow.
+  Workflow module irisworkflow built successfully.
+
+Because our workflow ``irisworkflow.speciesLength`` has not been connected to a source or a sink, is is technically a method and is treated as such. This means we can see it in alongside our other methods:
+
+.. code :: bash
+  
+  ~/irisworkflow/ $ nstack list methods
+  irisclassify.predict : (Double, Double, Double, Double) -> Text
+  demo.numChars : Text -> Integer
+  irisworkflow.speciesLength : (Double, Double, Double, Double) -> Integer
+
+Note that the input type of the workflow is the input type of ``irisclassify.predict``, and the output type is the output type of ``demo.numChars``. Like other methods, this can be connected to a source and a sink to make it `fully composed`:
+
+.. code :: bash
+
+  ~/irisworkflow/ $ nstack start "source(http:///speciesLength : (Double, Double, Double, Double)) | irisworkflow.speciesLength | sink(log:// : Integer)"
+
+Alternatively, you can move the source and sink into the ``workflow.nml`` file:
+
+.. code :: java
+
+  module irisworkflow {
+    def completeWorkflow = source(http:///speciesLength : (Double, Double, Double, Double)) | irisworkflow.speciesLength | sink(log:// : Integer)
+  }
+
+If you ``build`` this, you can then start it by itself with the ``start`` command, because it's a fully composed:
+
+.. code :: bash
+
+  ~/irisworkflow/ $ nstack start irisworkflow.completeWorkflow
+
+This paradigm can be helpful when we apply it to sources and sinks. Oftentimes, you -- or someone else in your company -- will want to create sources and sinks which are combined with modules, for instance in the following fictional example:
```

Comment:
Maybe we should build a practical example here that someone can go through, for defining sources and sinks?

E.g. define a reusable `http` endpoint:

```
def irisEndpoint = source(http:///irisclassify : (Double,Double,Double,Double))
```

Which they can then use and test? Or we stick with Postgres but make sure we actually have a test db on the server they can use.